### ISSUE TITLE
createvolume default AMI

### ISSUE THREAD

**Issue by nehalecky** [jtriley/StarCluster#173]
Thanks for a great package. StarCluster is grand. I was following the coherent manual quite swimmingly, but stumbled following the steps for [creating and formatting a new EBS volume](http://star.mit.edu/cluster/docs/latest/manual/volumes.html#create-and-format-a-new-ebs-volume).

I am currently using US West (N. California) region, in the most-hip zone `us-west-1c`. Entering the `createvolume` command, modified to reflect this as:
`$ starcluster createvolume --name=data 10 us-west-1c`

results in an `image does note exist` error.

```
starcluster createvolume --name=data 10 us-west-1c
StarCluster - (http://star.mit.edu/cluster) (v. 0.9999)
Software Tools for Academics and Researchers (STAR)
Please submit bug reports to starcluster@mit.edu

>>> No keypair specified, picking one from config...
>>> Using keypair: ec2-West-CA
!!! ERROR - Failed to create new volume
!!! ERROR - image ami-899d49e0 does not exist
```

This AMI is the default on the US East (Virginia) region, but, per my defaults specified in the starcluster config file, I'm not even located there! I've already modified and created my own AMI from the existing public starcluster AMI on US West (ami-3abd9b7f ), and was finally able to create a volume by specifying my personal AMI as:
`starcluster createvolume -i ami-3abd9b7f -n data 10 us-west-1c`

_Suggestion_: when creating volumes, the default logic for searching AMIs (when none are specified) should look to public starcluster AMIs in the region specified in starcluster config. 

Thank you for the great work! Godspeed.

**Comment by jtriley** [comment]
@nehalecky Sorry for the delay - you're absolutely right about searching for public SC AMIs in the target region. I plan to implement this but not before the next release given that I'm waaaay behind on a release. Thanks for reporting!

**Comment by nehalecky** [comment]
Thanks for the reply, @jtriley. I wanted to try and get you a solution myself, but just haven't had the time t dig into StarCluster much yet.  I'm hoping that this changes, soon. Thanks, again, for the great package.

**Comment by nitecoder** [comment]
This still happens in the released code (not sure about dev branch). But, there is a fairly easy workaround. You need to create a cluster with name "volumecreator" in the right region (probably also need to make sure there is a node in the right zone). Then when the createvolume runs, it will detect this as an already existing volume host and will use node from this cluster instead of trying to create a new one with the wrong AMI. Resulting output:

```
....
>>> No keypair specified, picking one from config...
>>> Using keypair: xxxxx-us-west
>>> Using existing instance i-852xxxxx in group @sc-volumecreator
>>> Waiting for instance i-852xxxxx to come up... 
>>> Checking for required remote commands...
>>> Creating 100GB volume in zone us-west-1a
>>> New volume id: vol-e67xxxx
....
```

**Comment by jtriley** [comment]
@nitecoder That's true but shouldn't be necessary - you can use the `-i` option to to the `createvolume` command to change the AMI used when launching a volume host, e.g.:

```
$ starcluster createvolume -i ami-3abd9b7f 10 us-west-1c
```
